# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

StatusActionGoal action_goal
StatusActionResult action_result
StatusActionFeedback action_feedback
